docs/monitoring/monitoring-kibana.asciidoc

```diff
@@ -15,9 +15,34 @@ method, see <<monitoring-metricbeat>>.
 To learn about monitoring in general, see 
 {stack-ov}/xpack-monitoring.html[Monitoring the {stack}]. 
 
-. Enable the collection of monitoring data in {stack}. Set 
-`xpack.monitoring.collection.enabled` to `true` on the production cluster.  +
-+ 
+
+. Enable the collection of monitoring data. 
+
+** In {kib}:
+
+... Open {kib} in your web browser. If you are running {kib} locally, go to 
+`http://localhost:5601/`. 
++
+--
+////
+If the Elastic {security-features} are enabled on the {kib} server, to access 
+{kib} and view the monitoring dashboards, you must log in as a user who has the 
+`kibana_user` and `monitoring_user` roles. For more information, see 
+{stack-ov}/built-in-roles.html[Built-in roles].
+//TBD: What permissions are required to update cluster settings?
+
+If you are accessing a remote monitoring cluster, you must log in to {kib}
+with credentials that are valid on both the {kib} server and the monitoring 
+cluster.
+////
```

Comment:
Did you mean to leave this commented-out section here?

Reply:
Yes, I have to test still to find out what credentials are required.

Reply:
Understood. Once you've done the testing and addressed this section accordingly, let me know and I'll re-review this PR then.

Reply:
I've finished some extensive changes and refreshed the preview server.